GenumKit/Stencil/Contexts.swift

```diff
@@ -80,13 +83,17 @@ extension StoryboardParser {
       if let scenes = storyboardsScenes[storyboardName] {
         sbMap["scenes"] = scenes
           .sort({$0.storyboardID < $1.storyboardID})
-          .map { (scene: Scene) -> [String:String] in
-            // Handle special scene.tag cases like navigationController, splitViewController, etc…
-            // TODO: Fix this to extract the 'UI' prefix and be able to implement OSX support in #128
-            let customClass = scene.customClass ??
-              (scene.tag != "viewController" ? "UI" + uppercaseFirst(scene.tag) : nil)
-            if let customClass = customClass {
-              return ["identifier": scene.storyboardID, "class": customClass]
+          .map { (scene: Scene) -> [String:AnyObject] in
+            if scene.customClass == nil && scene.tag == "viewController" {
```

Comment:
Just realized that this condition is the same as the next on and both could be grouped on only one condition:

```swift
 if scene.customClass == nil {
  return [
    "identifier": scene.storyboardID,
    "baseType": uppercaseFirst(scene.tag)
    "isBaseViewController": (scene.tag == "viewController")
  ]
}
```